functions/Remove-DbaDbData.ps1

```diff
@@ -0,0 +1,166 @@
+function Remove-DbaDbData {
+    <#
+    .SYNOPSIS
+        Removes all the date from a database(s) for each instance(s) of SQL Server.
+
+    .DESCRIPTION
+        This command truncates all the tables in a database. If there are foreign keys and/or views they are scripted out, then dropped before the truncate, and recreated after.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternate Windows or SQL Login Authentication. Accepts credential objects (Get-Credential).
```

Comment:
This should match the parameter help from other commands (consistency 😉 )

Reply:
Fixed these - I copied this from another function instead of the template - I'll find it and fix that one too. Thanks